src/strings/PowerFxResources.en-US.resx

```diff
@@ -3621,6 +3621,17 @@
     <value>https://go.microsoft.com/fwlink/?linkid=2132570</value>
     <comment>{Locked}</comment>
   </data>
+  <data name="ErrorResource_ErrBehaviorFunctionInDataUDF_ShortMessage" xml:space="preserve">
+    <value>Behavior function in a non-behavior user-defined function. Please wrap user-defined function with {} to declare a behavior UDF.</value>
```

Comment:
>wrap user-defined function with {} [](http://example.com/codeflow?start=77&length=34)

Suggestion: "Please wrap **the** user-defined function **body with curly braces ({...})** to declare..."